src/test/scala/scorex/transaction/BurnTransactionSpecification.scala

```diff
@@ -21,4 +25,69 @@ class BurnTransactionSpecification extends PropSpec with PropertyChecks with Mat
     }
   }
 
+  property("JSON format validation for BurnTransactionV1") {
+    val js = Json.parse("""{
+                       "type": 6,
+                       "id": "Ci1q7y7Qq2C2GDH7YVXsQ8w5vRRKYeoYTp9J76AXw8TZ",
+                       "sender": "3N5GRqzDBhjVXnCn44baHcz2GoZy5qLxtTh",
+                       "senderPublicKey": "FM5ojNqW7e9cZ9zhPYGkpSP1Pcd8Z3e3MNKYVS5pGJ8Z",
+                       "fee": 100000000,
+                       "timestamp": 1526287561757,
+                       "signature": "uapJcAJQryBhWThU43rYgMNmvdT7kY747vx5BBgxr2KvaeTRx8Vsuh4yu1JxBymU9LnAoo1zjQcPrWSuhi6dVPE",
+                       "chainId": null,
+                       "version": 1,
+                       "assetId": "9ekQuYn92natMnMq8KqeGK3Nn7cpKd3BvPEGgD6fFyyz",
+                       "amount": 10000000000
+                    }
+    """) //TODO: change to "quantity" after NODE-765 fix
+
+    val tx = BurnTransactionV1
+      .create(
+        PublicKeyAccount.fromBase58String("FM5ojNqW7e9cZ9zhPYGkpSP1Pcd8Z3e3MNKYVS5pGJ8Z").right.get,
+        ByteStr(Base58.decode("9ekQuYn92natMnMq8KqeGK3Nn7cpKd3BvPEGgD6fFyyz").get),
+        10000000000L,
+        100000000L,
+        1526287561757L,
+        ByteStr(Base58.decode("uapJcAJQryBhWThU43rYgMNmvdT7kY747vx5BBgxr2KvaeTRx8Vsuh4yu1JxBymU9LnAoo1zjQcPrWSuhi6dVPE").get)
+      )
+      .right
+      .get
+    js shouldEqual tx.json()
+  }
+
+  property("JSON format validation for BurnTransactionV2") {
+    val js = Json.parse("""{
+                       "type": 6,
+                       "id": "6QA1sLV53euVCX5fFemNuEyRVdQ5JYo5dWDsCmtKADRc",
+                       "sender": "3N5GRqzDBhjVXnCn44baHcz2GoZy5qLxtTh",
+                       "senderPublicKey": "FM5ojNqW7e9cZ9zhPYGkpSP1Pcd8Z3e3MNKYVS5pGJ8Z",
+                       "fee": 100000000,
+                       "timestamp": 1526287561757,
+                       "proofs": [
+                       "3NcEv6tcVMuXkTJwiqW4J3GMCTe8iSLY7neEfNZonp59eTQEZXYPQWs565CRUctDrvcbtmsRgWvnN7BnFZ1AVZ1H"
+                       ],
+                       "chainId": 84,
+                       "version": 2,
+                       "assetId": "9ekQuYn92natMnMq8KqeGK3Nn7cpKd3BvPEGgD6fFyyz",
+                       "amount": 10000000000
+                    }
+    """) //TODO: change to "quantity" after NODE-765 fix
+
+    val tx = BurnTransactionV2
+      .create(
+        2,
+        'T',
+        PublicKeyAccount.fromBase58String("FM5ojNqW7e9cZ9zhPYGkpSP1Pcd8Z3e3MNKYVS5pGJ8Z").right.get,
+        ByteStr(Base58.decode("9ekQuYn92natMnMq8KqeGK3Nn7cpKd3BvPEGgD6fFyyz").get),
```

Comment:
You can use `ByteStr.decodeBase58(…)` instead of `ByteStr(Base58.decode(…))`. Suggest to use replace regex in IDEA:
`ByteStr\(Base58\.decode\("(\w+)"\).get\)`
->
`ByteStr.decodeBase58("$1")`

Reply:
to ByteStr.decodeBase58("$1").get